src/libespeak-ng/event.c

```diff
@@ -45,8 +45,8 @@ static pthread_cond_t my_cond_stop_is_acknowledged;
 static bool my_stop_is_acknowledged = false;
 static bool my_terminate_is_required = 0;
 // my_thread: polls the audio duration and compares it to the duration of the first event.
-static pthread_t my_thread;
-static bool thread_inited;
+static pthread_t my_thread = 0;
```

Comment:
No, the type of `pthread_t` is not specified, one cannot assume that it can be set to 0 (and not even assume that 0 is a non-thread-id value)